index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link href="styles.css" rel="stylesheet"> 
+</head>
+<body>
+    <header>
+        <a href="index.html"><img src="media/Logo.svg" alt="Logo" class="logo"></a>
+        <nav>
+            <a href="index.html"><img src="media/Union.svg" alt="menu icon" class="hamburger"></a>
+        </nav>
```

Comment:
- [ ] In bigger mobile screens the `humburger` button is getting close to the center. Please consider using `flexbox` to justify content properly. It should always stay on the right side despite the mobile screen size. 
Current state: 
<img width="482" alt="Screenshot 2022-02-11 at 21 32 49" src="https://user-images.githubusercontent.com/77038610/153640346-92cfa559-a6cd-41d2-8218-4fd956bfe34b.png">